notebooks/01_data_acquisition.ipynb

```diff
@@ -98,7 +98,7 @@
    "cell_type": "markdown",
```

Comment:
<p>This is another test comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kamballu/notebook-ml/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>